src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -139,11 +140,67 @@
             "Id": "InstanceType",
             "Name": "EC2 Instance Type",
             "Description": "The EC2 instance type of the EC2 instances created for the environment.",
-            "Type": "String",
+            "Type": "Object",
```

Comment:
This is a breaking change to any existing users. I know we are in preview but we should give this serious thought on whether we should be making breaking changes.

Reply:
Reverted